Summary and industry list

Nebhey opened the pull request:
I add the industry list and the summary of this week. I am going to try to answer to the question at the end. If you could tell me if I am going in the wrong way, I would really appreciate (I am new in matter of research and this is a new soft skill so if I am terrible, I am terrible).

kbuzard commented on Summary of the 02.04.15 session.md:
```diff
@@ -0,0 +1,22 @@
+  * start reading the second book, _Traders and diplomats_ and the third book _The GATT legal system and World Trade Diplomacy_ by Robert Hudec
+  * some questions need to answered in order to define the  role of the TAA in the definition of the background of the model:
+      * Is the TAA relevant in multilateral negotiations like the GATT? Is the extension of the TAA linked to the negotiation at the GATT? If yes, how? 
+      * How does the TAA exactly condition the president action when negotiating? Does it have any resemblance with the fast-track procedures in the 2000s?
```
These questions look good: I would just make this second more more specific, or perhaps add this as a sub-question: what are some examples (no need to be exhaustive; that is, no need to find this for each renewal of TAA) of the parameters the TAA authorized the president to negotiate within. This would help to fix ideas in my mind.

kbuzard commented on Industry list.Rmd:
```diff
@@ -0,0 +1,54 @@
+
+Agriculture
+============
+  * 1934, when voting the TAA for the first time;at the initiative of the Republicans, in House and by Senator Overton, supported by a minority of Democrats. Concern the production stipulated in the AAA: __sugar__, __coconut oil__, __sesame__ __oil__, __laces__, __braids__, __wool__ and __wheat__. The amendments aim applying the cost-production calculus for these particular product and spare them ...
```
More natural English is "The goal of the amendments is to require cost-production calculus be applied for these particular product and to spare them from tariff cuts. The amendment is defeated even though it was supported by some Democrats.

kbuzard commented on Industry list.Rmd:
```diff
@@ -0,0 +1,54 @@
+============
+  * 1934, when voting the TAA for the first time;at the initiative of the Republicans, in House and by Senator Overton, supported by a minority of Democrats. Concern the production stipulated in the AAA: __sugar__, __coconut oil__, __sesame__ __oil__, __laces__, __braids__, __wool__ and __wheat__. The amendments aim applying the cost-production calculus for these particular product and spare them ...
+  * 1937, when voting the first extension of the TAA, introduced in the Senate by the Republicans and followed by a lot of Democrats (20 in Senate). The amendments aim applying the cost-production calculus for agricultural products and spare them from tariff cuts.The amendment is first adopted then rejected in the final vote.
+  * 1953, when voting an extension of the TAA, brought by Senator Magnuson (D), Cordon (R) and Representative Simpson modified for the House Ways and Means Committee under the presidency of Chairman Reed. The amendments forbid new tariff cuts on agricultural product (especially __wool__), enable rising tariffs by use of the the Section 22 of the AAA on the TAA. Adopted.
```
"use of Section 22" doesn't require article "the" before "Section 22"

kbuzard commented on Industry list.Rmd:
```diff
@@ -0,0 +1,54 @@
+
+Defense
+============
+  * 1954, when voting an extension of the TAA. Brought by Senator Symington (Democrat). It stipulates that no action should be taken in the decrease of a tariff if it appears to threaten meeting the defense requirements. 
```
"It stipulates that no action should be taken to decrease a tariff if it appears to threaten meeting defense requirements."

kbuzard commented on Industry list.Rmd:
```diff
@@ -0,0 +1,54 @@
+
+Fishing
+============
+  * 1949 when voting an extension of the TAA ; Senator Milliken, senior Republican in charge of the foreign policy in the opposition;raising tariffs in order to protect the new importation. The amendment failed in the Senate by a partisan vote. 
```
don't need "the" before "foreign policy"

I don't know what "to protect the new importation" means

kbuzard commented on Industry list.Rmd:
```diff
@@ -0,0 +1,54 @@
+============
+  * 1949 when voting an extension of the TAA ; Senator Milliken, senior Republican in charge of the foreign policy in the opposition;raising tariffs in order to protect the new importation. The amendment failed in the Senate by a partisan vote.
+  * 1953, when voting an extension of the TAA, brought by Representative Simpson modified for the House Ways and Means Committee under the presidency of Chairman Reed. implementation of quotas on crude oil and residual fuel oil. 
+  * 1955, when voting an extension of the TAA, brought by Senator Neely. It implements a 10% quota on oil and residual fuel. It is not accepted but amended into a amendment regarding national security industry that enable the President to raise the level of tariff. This one is adopted. 
```
instead of amended into an amendment, I would say "converted" into an amendment just to avoid redundancy. enable should be enables. This should perhaps also be entered under the defense industry? I'm not sure...

Nebhey replied:
I think that even if the argument is about defense, the target of the amendment was to protect oil. It is more to protect a domestic industry than to protect the defense industry. Under the section section defense, I would only put things that directly concern security. However, I agree that for strategic industries, security and economics are tightly linked.

kbuzard commented:
This is a great start to the industry list. As you go forward, please see if you can add any details about the efforts of the industry groups related to the legislative actions.

kbuzard commented:
I don't think it's necessary to get into details about the levels. I was just hoping for a general clarification about whether they were concerned with preventing increases in imports on goods that were already imported, or on new tariff lines (this language is used in several places throughout the document)

Nebhey commented:
Hello Professor, I updated the merge, I added the summary of the session of yesterday. I hope I can finish the third book for next week and have the data (some). Have a good weekend!

Nebhey commented:
Dear Professor, 
I uploaded the data, as you can see, there is not that much therefore, I think I can add countries that negotiated during several rounds: Haiti, Norway, Peru, South Africa, Sweden (3 rounds of negotiation in bilateral).

kbuzard commented:
Let me merge in this data and take a look at it and we can make decisions about how to move forward with the data collection when we meet this afternoon.
